cmd/evm/blockrunner.go

```diff
@@ -79,6 +96,11 @@ func runBlockTest(ctx *cli.Context, fname string) ([]testResult, error) {
 	}
 	tracer := tracerFromFlags(ctx)
 
+	// Suppress INFO logs when tracing to avoid polluting stderr
+	if tracer != nil {
```

Comment:
Not sure if it's a good idea to completely disable the log. 
The condition is pretty weird, does the command emit a tons of logs with tracer enabled?

An alternative will be exposing the `internal/debug/Flags` to the users, letting them to configure the log instead. I am not sure if it's worthwhile.

Reply:
Fixed! goevmlab/blocktest fuzzer can use evm blocktest --trace test.json --verbosity 0  to disable the rare logs to stderr when there are errors to cleanly parse the trace json lines.